CONTRIBUTING.md

```diff
@@ -153,6 +154,15 @@ For overrides of virtual methods the `copydoc` tag can be used:
 
 A good example for code formatting and doxygen structure can be found in [swe_docu_guidelines.md (WIP)](./doc/aspice_swe3_4/swe_docu_guidelines.md)
 
+### External dependencies
+
+External dependencies such as the [STL](https://en.wikipedia.org/wiki/Standard_Template_Library) or
+other libaries shall be kept to a minium for `iceoryx_posh` and `iceoryx_hoofs`. If you think a new dependency is
+necessary, do the following:
+
+1. Engage with the maintainers beforehand by opening an issue to discuss the necessity
```

Comment:
This sounds a little bit weird. How about: Talk with the maintainers ... or Contact the maintainers beforehand...